llvm/lib/MC/MachObjectWriter.cpp

```diff
@@ -1103,10 +1103,72 @@ void MachObjectWriter::writeSymbolTable(MCAssembler &Asm,
     StringTable.write(W.OS);
 }
 
+void MachObjectWriter::applyAddends(MCAssembler &Asm,
+                                    const MCAsmLayout &Layout) {
+  for (MCSection &Sec : Asm) {
+    for (MCFragment &F : Sec) {
+      switch (F.getKind()) {
+      default:
```

Comment:
There aren't that many more fragment types, so it's best to explicitly list them here. If we use a default case then we don't get an uncovered-switch warning from clang when a new enumerator gets added.